consultation_analyser/consultations/views/filters.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models import Count
+from django.http import HttpRequest
+
+from .. import models
+
+
+def get_applied_filters(request: HttpRequest):
+    return {
+        "keyword": request.GET.get("keyword", ""),
+        "theme": request.GET.get("theme", "All"),
+        "opinion": request.GET.get("opinion", "All"),
+    }
+
+
+def get_filtered_responses(question: models.Question, applied_filters):
```

Comment:
Could have missed them, but I think you could add a couple of tests for this function and `get_filtered_themes`.